### PR TITLE
[CP-941] fix getCrashDump event

### PR DESCRIPTION
**Pull request by wercix**
Jira: [CP-941]

**Description**
getCrashDump event was accidentially removed 

<details>
<summary><b>Screenshots</b></summary>
// put images here
</details>

[CP-941]: https://appnroll.atlassian.net/browse/CP-941?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/app/src/renderer/wrappers/root-wrapper.tsx [hunk 1]
```diff
@@ -148,6 +148,19 @@ const RootWrapper: FunctionComponent<Props> = ({
     connect()
   }, [])
 
+  useEffect(() => {
+    let interval: NodeJS.Timeout
+
+    if (connectedAndUnlocked) {
+      getCrashDump()
+      interval = setInterval(() => updateAllIndexes(), 30000)
```
Comment (dkarski) [inline packages/app/src/renderer/wrappers/root-wrapper.tsx:156]:
```suggestion
```

Comment (igorbezsmertnyi) [reply packages/app/src/renderer/wrappers/root-wrapper.tsx:156]: Maybe a feature flag would be better in this case

**Review by dkarski — APPROVED** [review]

File: packages/app/src/renderer/wrappers/root-wrapper.tsx (hunk 1 above)
Comment (dkarski) [reply packages/app/src/renderer/wrappers/root-wrapper.tsx]: @igorbezsmertnyi what do you mean? You would like to add a feature flag for crashDump logic? 🤔

Comment (igorbezsmertnyi) [reply packages/app/src/renderer/wrappers/root-wrapper.tsx]: @dkarski I mean this line `interval = setInterval(() => updateAllIndexes(), 30000)`

Comment (dkarski) [reply packages/app/src/renderer/wrappers/root-wrapper.tsx]: this line is deprecated 😛

**Review by igorbezsmertnyi — APPROVED** [review]

**Pull request merged**